packages/cli/src/commands/models/create.ts

```diff
@@ -80,15 +66,20 @@ export function getCreateCommand(): CLICommand {
         if (featureFlagManager.getBooleanValue(FeatureFlags.CLIDotNet)) {
           // this feature is used in e2e test to scaffold VS project in non-interactive mode
           inputs.platform = Platform.VS;
+          inputs["template-name"] = inputs.capabilities;
+          inputs["programming-language"] = "csharp";
```

Comment:
With this, user will only be able to scaffold csharp templates once enabled toggle. Is that expected?

Reply:
Yes, this is the current UX, please see line67 comment